nix/pkgs/mayastor/default.nix

```diff
@@ -36,8 +37,7 @@ let
             lib.hasPrefix (toString (src + "/${allowedPrefix}")) path)
           allowedPrefixes)
       src;
-  version_drv = import ../../lib/version.nix { inherit lib stdenv git; };
-  version = builtins.readFile "${version_drv}";
+  version = (builtins.fromTOML (builtins.readFile ../../../mayastor/Cargo.toml)).package.version;
```

Comment:
the version in the toml never changes, not sure if this will cause any issues

Reply:
Really!? ... Why?

Reply:
Dunno, I guess no one ever bothered to update it :(

Reply:
#754 tracks this! Thanks for raising it!